homeassistant/components/meteoalarm/binary_sensor.py

```diff
@@ -1,11 +1,11 @@
-"""Binarysensor for MeteoAlarm.eu."""
+"""Binary sensor for MeteoAlarm.eu."""
 from datetime import timedelta
 import logging
 
 import voluptuous as vol
 
 from homeassistant.components.binary_sensor import (
-    PLATFORM_SCHEMA, BinarySensorDevice)
+    PLATFORM_SCHEMA, binary sensorDevice)
```

Comment:
Oh, this is wrong.

Reply:
Yes, stupid mistake. I have search and replaced it Oops ;)